docs/developer-guide/api-user-guide.md

```diff
@@ -0,0 +1,255 @@
+
+For comprehensive details of all zot API commands, see [Viewing the complete zot API reference](#_find_reference).
+
+The zot API implements the [OCI Distribution endpoints](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#endpoints) along with additional endpoints for supported extensions. You can access the REST API at the same URL and port number used by the GUI and by datapath tools.
+
+> :pencil2: The examples in this article assume that the zot registry is located at `localhost:8080`.
+
+
+## Supported API endpoints
+
+The following is a list of zot API endpoints along with the conditions under which each endpoint is available.
+
+> :pencil2: Some API endpoints are available only when a specific extension is enabled in the zot configuration file, or when an extension build label is specified in the `make` command (for example, `make binary EXTENSIONS=ui`), or both.
+
+For comprehensive details of these zot API endpoints, see [Viewing the complete zot API reference](#_find_reference).
+
+| Endpoint | Actions | Description | Availability |
+| --- | ------------ | -------- | --------|
+| / |  | Web interface | Enabled by using the `ui` build label and enabling the `ui` extension in the configuration file. |
+| /auth/apikey| DELETE, GET, POST | Creates, lists, or deletes API keys | Available when API key authentication is enabled in the configuration file ("apikey": true). |
+| /auth/logout| POST  | Ends an API session | Available when authentication is available. This includes not only OpenID, but all session-based authentication. |
+| /oras/artifacts/v1/_zot/ \ manifests/{digest}/referrers| GET | [OCI Registry As Storage (ORAS)](https://oras.land/) endpoints | Always enabled. |
+| /v2/| GET | [OCI specification endpoints](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#endpoints) | Always available. |
+| /v2/_catalog| GET |  |
+| /v2/_oci/ext/discover| GET | Discover extensions per the OCI specification | Always available. |
+| /v2/_zot/ext/cosign| POST | Uploads keys for signature verification | Enabled by using the `imagetrust` build label and enabling the `trust` extension with the `cosign` option enabled. |
+| /v2/_zot/ext/mgmt| GET | Mgmt extension endpoints | Enabled by using the `mgmt` build label and enabling the `search` extension in the configuration file. |
+| /v2/_zot/ext/notation| POST | With query parameters, uploads certificates for signature verification | Enabled by using the `imagetrust` build label and enabling the `trust` extension with the `notation` option enabled. |
+| /v2/_zot/ext/search |  | Enhanced search | Enabled by using the `search` build label and enabling the `search` extension in the configuration file. |
+| /v2/_zot/ext/userprefs| PUT | User preferences endpoints | Enabled by using the `userprefs` build label and enabling both the `search` and the `ui` extensions in the configuration file. |
+| /v2/_zot/blobs/{digest}| DELETE, GET, HEAD |  |
```

Comment:
These urls are not correct (lines 34-38)

Reply:
Ram's suggestion for escaping `/v2/manifests/\<\>/repo`

Reply:
These are the urls defined in the spec at : https://github.com/opencontainers/distribution-spec/blob/main/spec.md#endpoints

Reply:
Revised OCI endpoints section